README.md

```diff
@@ -69,40 +69,37 @@ Prerequisites
 
    **MacOS**:
    ```
-   $ brew tap riscv/riscv && brew update && brew install riscv-gnu-toolchain --with-multilib
+   $ brew tap riscv/riscv && brew update && brew install riscv-gnu-toolchain
    ```
-   Warning: this will compile from source, and takes a while. Also this will
-   build a bleeding-edge version of GCC, and there is a small chance it will not
-   work. However, we have successfully obtained a toolchain this way.
 
    **Ubuntu (19.10 or later)**:
    ```
    $ sudo apt install gcc-riscv64-unknown-elf
    ```
 
-   Unfortunately, Ubuntu does not currently (June 2020) provide a package for
+   Unfortunately, Ubuntu does not currently (June 2022) provide a package for
```

Comment:
I don't think this is true. Picolib seems to be packaged for Ubuntu now

Reply:
https://packages.ubuntu.com/jammy/picolibc-riscv64-unknown-elf ?

Reply:
Yeah!

Reply:
Added picolib.